### PR TITLE
add the auto watering example for haas solution #1671

### PR DESCRIPTION
**Pull request by wushengtws**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : ba1a334bf04035104fddeed7ad3a4d5f122c856b

**Comment by skylarCai** [comment]
auto build success,number:138

**Comment by skylarCai** [comment]
autotest success,number:1348

File: haas_lib_bundles/python/docs/examples/auto_watering/esp32/README.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
+# 自动浇水养植系统
```
Comment (skylarCai) [inline haas_lib_bundles/python/docs/examples/auto_watering/esp32/README.md:1]: nice

**Review by skylarCai — COMMENTED** [review]
nice

**Review by ethanlcz — APPROVED** [review]

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**